_data/sites.json

```diff
@@ -4106,6 +4183,26 @@
         ]
     },
 
+    {
+        "name": "MacTrade",
+        "url": "https://www.mactrade.de/goodahead_customer/account/delete/",
```

Comment:
Does the link delete the account directly? Just asking as there are no notes

Reply:
Thank you for the question. The link does not delete the account immediately. You still need to confirm the deletion. I added notes to clarify.

Reply:
Thanks!